src/message.rs

```diff
@@ -1236,6 +1236,9 @@ pub async fn delete_msgs(context: &Context, msg_ids: &[MsgId]) -> Result<()> {
             .trash(context)
             .await
             .with_context(|| format!("Unable to trash message {}", msg_id))?;
+
+        context.emit_event(EventType::WebXdInstanceDeleted { msg_id: *msg_id });
```

Comment:
currently, this is emitted for all messages types, it should be emitted only for `Viewtype::Webxdc`.